### ISSUE TITLE
System with Multiple Network Interfaces

### ISSUE THREAD

**Issue by kitgul** [nmaier/simpleDLNA#40]
In a system with multiple network interfaces it would be nice to have the option to chose wich one to use (ip setting of the server)

**Comment by Stannieman** [comment]
Yes this is really a problem. Not so much for the non tech savvy users, but for me for example it constantly binds to internal network adapters for Hyper-V or Docker. It seems to just bind to the first IP it can find. The project seems to be quite inactive so I'll see if I can make it work for at least the command line app somewhere this week.

As far as I can see the problem is here https://github.com/nmaier/simpleDLNA/blob/master/server/Http/HTTPServer.cs#L56
It should be possible to pass a different System.Net.IPAddress instance to the constructor just like is done with the port.

**Comment by Stannieman** [comment]
Ok we mighte be better off looking at a fork. This does not build out of the box cause it tries to copy some files to D:\bin (why???) which doesn't exist on my system, and it's not so obvious where that's coming from. So I think first some modernization should be done before implementing new features, think of moving to Dotnet Core, use newer installer, get rid of NuGet.exe in repo (why was it ever put there?).
Oh, and there are no unit tests so it's easy to break existing functionality without realizing it. Especially this part makes me feel not very comfortable to work on it. In a code base you're not familiar with it's too easy to make mistakes.
